src/main/java/slimeknights/tconstruct/smeltery/TinkerSmeltery.java

```diff
@@ -237,6 +239,7 @@ public final class TinkerSmeltery extends TinkerModule {
   public static final ItemObject<ChannelBlock> searedChannel = BLOCKS.register("seared_channel", () -> new ChannelBlock(SEARED_NON_SOLID), TOOLTIP_BLOCK_ITEM);
   public static final ItemObject<CastingBasinBlock> searedBasin = BLOCKS.register("seared_basin", () -> new CastingBasinBlock(SEARED_NON_SOLID, false), TOOLTIP_BLOCK_ITEM);
   public static final ItemObject<CastingTableBlock> searedTable = BLOCKS.register("seared_table", () -> new CastingTableBlock(SEARED_NON_SOLID, false), TOOLTIP_BLOCK_ITEM);
+  public static final ItemObject<CastingTankBlock> castingTank = BLOCKS.register("casting_tank", () -> new CastingTankBlock(SEARED_NON_SOLID, PushReaction.DESTROY), b -> new TankItem(b, ITEM_PROPS, true));
```

Comment:
Naming convention for the fields would make this `seared_casting_tank`/`searedCastingTank`. Not because we are definitely adding it, but to avoid needing to rename if we do.

Has the bonus of the block entity field can be just `castingTank`.